metadata-ingestion/src/datahub/telemetry/telemetry.py

```diff
@@ -42,6 +42,16 @@ def __init__(self):
         else:
             self.load_config()
 
+        # send updated user-level prop{erties
+        mp.people_set(
```

Comment:
Seems like this line is by-passing the `self.enabled` flag.  The code should check that flag before calling mixpanel.  I have updated my fork to `HEAD` and ran into a proxy issue (my org is blocking mixpanel).

Reply:
@kevinhu @shirshanka See my comment above.

Reply:
My bad—I have a fix out at https://github.com/linkedin/datahub/pull/4253